industrial_taxonomy/getters/text_sectors.py

```diff
@@ -0,0 +1,30 @@
+"""Data getters for topsbm outputs"""
+
+from metaflow import Run
+from typing import Dict, List, Optional, Tuple
+
+from industrial_taxonomy.utils.metaflow import get_run
+from industrial_taxonomy.pipeline.glass_clusters.sbmtm import sbmtm
+
+
+clustering_param = str
+sector_id = str
+glass_id = str
+
+
+def text_sectors(
+    run: Optional[Run] = None,
+) -> Dict[clustering_param, List[Tuple[sector_id, glass_id]]]:
+    """Gets text sector assignments from the topic modelling"""
```

Comment:
I think it would be useful to have a longer docstring here (or somewhere in the flow?) to explain what the different `clustering_param` values mean/describe.